src/modules/flow/iio/iio.json

```diff
@@ -53,7 +53,7 @@
          },
          {
            "data_type": "string",
-           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
+           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. Set to hrtimer:<trigger name> if want to use hrtimer trigger. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
```

Comment:
If it's intentional to allow `hrtimer:` only triggers, so Soletta can create an 'anonymous' hrtimer trigger, please explain this on docs, here and on `sol-iio.h`. But is it worth? Do you see this as a common case?